eml/screens/courses/CourseScreen.jsx

```diff
@@ -66,6 +67,10 @@ export default function CourseScreen() {
         return update;
     }, [navigation]);
 
+    useEffect(() => {
```

Comment:
As I read it we're showing this message every time we enter the page?
Maybe not if the page is constantly kept loaded in the background? idk

Reply:
It does not show the message again when renavigating to CourseScreen. I however do agree that this can be improved on in the future.